hub/management/commands/import_council_rural_classification.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import add_gss_codes, filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+shades = [
+    {"title": "Sparse and rural", "shader": ""},
+    {"title": "Rural", "shader": ""},
+    {"title": "Urban", "shader": ""},
+    {"title": "Urban with rural areas", "shader": ""},
+]
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council rural/urban classification"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Urban Rural Classification",
```

Comment:
Despite the wording on data.mysociety.org, I think this should _actually_ be “Rural Urban Classification” (at least, that’s what ONS and Government both use).